docs/containers-storage-mounted.md

```diff
@@ -0,0 +1,17 @@
+## containers-storage-mounted 1 "November 2018"
+
+## NAME
+containers-storage mounted - Check if a file system is mounted
+
+## SYNOPSIS
+**containers-storage** **mounted** *LayerOrContainerNameOrID*
```

Comment:
Should this be "LayerOrContainerNameOrID" or "layerOrContainerMountpointOrNameOrID"?  @giuseppe or @nalind do you know?

Reply:
The current logic doesn't attempt to look up a layer by mount point, so _LayerOrContainerNameOrID_ looks like the correct description.